src/integration_tests/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ add_executable(integration_tests_runner
     camera_take_photo_interval.cpp
     camera_format.cpp
     camera_test_helpers.cpp
-    component_information.cpp
+    #component_information.cpp
```

Comment:
```suggestion
```

Reply:
Hmm, you caught me :smile:. No thanks for checking, I clearly missed this one!

Reply:
Ok I realize why I had this disabled: 
1. The component information protocol will change again in the mavlink spec, so this is work in progress anyway.
2. Also, the server side requires mavlink ftp, however, the mavlink ftp plugin has not been refactored and split into server and client to fit into system impl and server component impl respectively.

For now, I moved this test over to system_tests but I'll have to disable it for now, as it's not passing without FTP.